tests/test_paths.py

```diff
@@ -32,7 +32,6 @@ def test_001():
         w.set_target(i, i % 2, i + 1)
 
     p = Paths(w)
-    assert p.at_end()
```

Comment:
Just to say that I think this is a bug in libsemigroups, that the return value of `at_end()` is non-deterministic if the Paths object isn't properly initialised (it somehow isn't tracked in the class whether or not it is initialised, but it is implicitly assumed that it is).